### PR TITLE
feat(Experiments): Create New web experiment from dotcom UI

### PR DESCRIPTION
**Pull request by Phanatic**
## Problem

Create Web Experiment from the `New Experiment` page.

## Changes
image

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

Manual testing

### CODE REVIEW

**Comment by jurajmajerik** [comment]
Could we do this on the first page of the experiment form, along with some explanations? You could reuse the Experiment goal selector:

image

**Comment by Phanatic** [comment]
> https://github.com/PostHog/posthog/pull/26009#issuecomment-2457667149
Sounds great! I'll include that too.

**Comment by jurajmajerik** [comment]
Please match the font size to the one already used in the Goal type selector :)

image

image

File: frontend/src/scenes/experiments/ExperimentForm.tsx [hunk 1]
```diff
@@ -48,6 +50,34 @@ const StepInfo = (): JSX.Element => {
...
+                                {
+                                    value: 'product',
+                                    label: 'Product experiment',
+                                    description: 'Use custom code to manage how variants modify your product.',
+                                },
+                                {
+                                    value: 'web',
+                                    label: 'No-code web experiment',
+                                    description:
+                                        'Define variants on your website effortlessly using the PostHog toolbar, no coding required.',
```
Comment (jurajmajerik) [inline frontend/src/scenes/experiments/ExperimentForm.tsx:75]: This is just my preference, but the word "effortlessly" sounds a bit marketing-y to me (there clearly _is_ some effort required), so I'd remove it :)

```suggestion
                                        'Define variants on your website using the PostHog toolbar, no coding required.',
```

**Review by jurajmajerik — APPROVED** [review]
Looks great, two nits :)

**Pull request merged**